test/server/cards/19.1-AS01/Dragon/KitsukiMasanori.spec.js

```diff
@@ -1,143 +1,219 @@
-describe('Kitsuki Masanori', function() {
-    integration(function() {
-        beforeEach(function() {
-            this.setupTest({
-                phase: 'conflict',
-                player1: {
-                    inPlay: ['mirumoto-raitsugu'],
-                    dynastyDiscard: ['kitsuki-masanori'],
-                    conflictDiscard: ['a-new-name', 'high-kick', 'duelist-training', 'writ-of-sanctification', 'shiba-s-oath'],
-                    hand: ['hidden-lineage', 'true-strike-kenjutsu', 'forebearer-s-echoes']
-                },
-                player2: {
-                    inPlay: ['iuchi-rimei'],
-                    hand:['let-go', 'fine-katana']
-                }
+describe('Kitsuki Masanori', function () {
```

Comment:
Make sure you add a test case for trying to discard the weapon when it's pulled out of discard